src/views/Folio.vue

```diff
@@ -9,51 +9,71 @@
       <div
         v-for="(image, index) in images"
         :key="index"
-        class="cf fl w-100 w-50-m w-25-ns dib"
+        class="cf fl w-100 w-50-m w-25-ns"
       >
         <FolioImage
           :src="image.path"
           :alt="image.alt"
         />
       </div>
     </div>
+    <FolioImageModal
+      v-if="showModal"
+      :src="modalImageParameters.path"
+      :alt="modalImageParameters.caption"
+      @open-modal-image="openModalImage"
+    />
   </div>
 </template>
 
 <script>
 import NavBar from '@/components/NavBar'
 import SocialBar from '../components/SocialBar'
 import FolioImage from '../components/FolioImage'
+import FolioImageModal from '../components/FolioImageModal'
 
 export default {
   name: 'Folio',
   components: {
+    FolioImageModal,
     FolioImage,
     SocialBar,
     NavBar
   },
   data () {
     return {
       images: [{
-        alt: 'Shade',
+        alt: 'alt',
```

Comment:
alt's are the text displayed if the image doesn't work . it would be a good idea to have them described by a name. you can keep the old names if they match the pictures

Reply:
That's fine, i know and i'll update them later with titles instead as those were just the characters' names